invokeai/frontend/web/src/features/nodes/util/graphBuilders/buildLinearTextToImageGraph.ts

```diff
@@ -58,12 +61,14 @@ export const buildLinearTextToImageGraph = (
    */
 
   // copy-pasted graph from node editor, filled in with state values & friendly node ids
+
+  // TODO: Actually create the graph correctly for ONNX
   const graph: NonNullableGraph = {
     id: TEXT_TO_IMAGE_GRAPH,
     nodes: {
-      [MAIN_MODEL_LOADER]: {
-        type: 'main_model_loader',
-        id: MAIN_MODEL_LOADER,
+      [model_loader]: {
+        type: model_loader,
```

Comment:
This is only accidentally valid. The `MAIN_MODEL_LOADER` constant is intended to be a human-readable name for the node. These constants are not guaranteed to also be valid node types. The type should be specified independently.